source/_data/glossary.yml

```
@@ -30,6 +30,11 @@
     Actions are used in several places in Home Assistant. As part of a script or
     automation, actions define what is going to happen once a trigger is
     activated. In scripts, an action is called *sequence*.
+- term: Actor
+  definition: |-
+    An entity that performs an action or initiates a changes in a system.
```

```suggestion
    An entity that receives a control signal and performs an action in a system.
```
should we add that it only does anything after receiving an input?